packages/familie-form-elements/form-elements.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { FamilieCheckbox } from './src';
+
+export default {
+    component: FamilieCheckbox,
+    parameters: {
+        componentSubtitle: 'Visittkort brukes til å vise enkel informasjon om en bruker.',
+    },
+    title: 'Komponenter/Visittkort',
```

Comment:
```suggestion
    title: 'Komponenter/FormElements/Checkbox',
```

Usikker på hvordan hiarkiet blir her, men vi ønsker Komponenter -> Form elements -> Checkbox